mmv1/products/apigee/terraform.yaml

```diff
@@ -114,6 +114,29 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       delete_minutes: 30
     custom_code: !ruby/object:Provider::Terraform::CustomCode
       custom_import: templates/terraform/custom_import/apigee_environment.go.erb
+  Envgroup: !ruby/object:Overrides::Terraform::ResourceOverride
+    autogen_async: true
+    import_format: ["{{org_id}}/envgroups/{{name}}", "{{org_id}}/{{name}}"]
+    examples:
+    - !ruby/object:Provider::Terraform::Examples
+      name: "apigee_environment_group_basic"
+      skip_test: true
+    - !ruby/object:Provider::Terraform::Examples
+      # This is a more verbose version of the above that creates all
+      # the resources needed for the acceptance test.
+      name: "apigee_environment_group_basic_test"
+      primary_resource_id: "apigee_environment_group"
+      test_env_vars:
```

Comment:
I think we want to add some special handling for the service networking connection. That resource causes problems for our CI. We want to bootstrap these networks for sharing across apigee org tests. See https://github.com/GoogleCloudPlatform/magic-modules/blob/master/mmv1/products/redis/terraform.yaml#L41 for an example

Reply:
The reason why we can't use a boostrapped VPC/service networking is that those need to be created on the ephemeral project where we host the parent Apigee org that owns this resource. Even if we were to have the VPC on a shared VPC on the integration host project, we'd need to set these ephemeral projects as service projects to the integration project.

So a better solution would be to create a bootstrapped Apigee Org, and not recreate the Org for every child resource we implement. We already have three: Instances, Environments and Environments groups, which should not be creating Orgs.  as part of their acceptance tests as that is a complex operation as the steps show.

We discussed this briefly [here](https://github.com/GoogleCloudPlatform/magic-modules/pull/4503#issuecomment-782049223) but unfortunately I had no time to look into this yet. Although I still want to do it, I think this shouldn't block the envgroups functionality here, since these tests are valid, just not ideal/optimized.